### DISCUSSION TITLE
Vanity Subdomain no longer working

### DISCUSSION THREAD

**Discussion by rileytaylor** [supabase/supabase#19881] [Questions] [answered]
We've been using a vanity subdomain successfully for months now, however, as of last night it seems to no longer be working at all.

Most of our apps and services have been using the vanity subdomain, so most of our stuff broke last night around midnight (US) when this change seems to have occurred. API requests are returning a `400` status with a message like "Project not specified" when made against the vanity subdomain.

~On the supabase dashboard, I can see it making requests to `https://api.supabase.io/v1/projects/<id>/custom-hostname`, which are also returning a 400 error: `{"message":"Project <id> is not allowed to set up custom domain; please contact Supabase support."}`~

~We aren't using a custom domain, so I don't know why it's returning that response.~ (Edit: Striking this out since it likely isn't relevant to the problem)

Using the cli I was able to verify that our vanity subdomain is still active, so it's not an issue there.

I've been able to move most of our stuff over to use the project url, but some things cannot be switched so easily: Auth being an example, where once you activate the subdomain, you can only access it via the subdomain [according to the CLI docs here](https://supabase.com/docs/reference/cli/supabase-vanity-subdomains-activate).

After this, I'm probably going to move away from the vanity subdomain, but also, this sudden breakage seems like it might be a bug from recent updates. I've already put in a support ticket, but I wanted to throw some info here in case this is causing issues for anyone else and to generate some awareness if that is the case.

**Comment by inian** [comment]
Hey @rileytaylor, can you please create a support ticket with your affected project at https://supabase.help so that we can investigate further? This might be a project specific issue and creating a support ticket from the dashboard helps with the investigation.

**Reply by rileytaylor** [reply]
Already did, just wanted to see if anyone else had the issue.

**Reply by inian** [reply, marked as answer]
I was able to confirm that there was an error with some projects which had enabled vanity subdomains. Thanks for flagging this Riley. 

The fix went out 10 minutes ago . If anyone is still experiencing this error, please raise a support ticket so that we can investigate.

**Comment by phamhieu** [comment]
> On the supabase dashboard, I can see it making requests to https://api.supabase.io/v1/projects/<id>/custom-hostname, which are also returning a 400 error:
> We aren't using a custom domain, so I don't know why it's returning that response.

Hello @rileytaylor I apologize for any confusion. The purpose of that endpoint is to verify the availability of a custom domain. It appears that the custom domain feature is not allowed for your project.

**Reply by rileytaylor** [reply]
Wasn't sure if it was relevant to the vanity subdomain or not. Either way, our vanity subdomain is enabled, but all requests through it are still failing.
